runtime/lua/vim/filetype.lua

```diff
@@ -1559,6 +1557,8 @@ local filename = {
   ['sshd_config'] = 'sshdconfig',
   ['/etc/sudoers'] = 'sudoers',
   ['sudoers.tmp'] = 'sudoers',
+  ['/sway/config'] = 'swayconfig',
+  ['/.sway/config'] = 'swayconfig',
```

Comment:
I think these should be removed, `/sway/config` and `/.sway/config` are not realistic absolute file paths (even though they *technically* match the pattern in `filetype.vim`). Same for the `i3` ones too.

The entries in the pattern section need to be updated too.

Reply:
Yeah, I was about to say. I'm not too fond of the magic initial `*` -- it's always better to be explicit.